pisa/analysis/analysis.py

```diff
@@ -1190,8 +1190,12 @@ def _minimizer_callable(self, scaled_param_vals, hypo_maker, data_dist,
             fit_history.append(
                 [metric_val] + [v.value.m for v in hypo_maker.params.free]
             )
-            
-        return sign*metric_val
+
+        penalty = 0.
+        if not external_priors_penalty is None:
+            penalty = np.abs(external_priors_penalty(hypo_maker, metric))
```

Comment:
why `abs` here? If we need absolute values, i think it should be done inside `external_priors_penalty`